### PR TITLE
Update to angular v14

### PR DESCRIPTION
**Pull request by izahirclemencia**

### CODE REVIEW

**Comment by TheFrohlich** [comment]
Hi, Is this going to be merged any time soon ?

**Comment by webcat12345** [comment]
After the check, hopefully tomorrow, I will merge and release the new version. Sorry guys for delay!

**Comment by TheFrohlich** [comment]
Thanks @webcat12345  - much appreciated and no apologies needed :)

**Comment by kevynsax** [comment]
Hi @webcat12345 any news about this merge?

**Comment by juvegitau** [comment]
> After the check, hopefully tomorrow, I will merge and release the new version. Sorry guys for delay!

Any news about  the merge @webcat12345 ? polite reminder

**Comment by mario-getpassport** [comment]
Hey @webcat12345, any news on this?

**Comment by mishevski8** [comment]
Hey @webcat12345, any news on this?

**Comment by farley911** [comment]
I too am looking forward to this update.

**Comment by guillermodelagala** [comment]
Please @webcat12345  , accept this PR.

Thanks in advance.

File: projects/ngx-intl-tel-input/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
 	"name": "ngx-intl-tel-input",
-	"version": "3.2.0",
+	"version": "3.3.0",
 	"peerDependencies": {
-		"@angular/common": "8.x - 13.x",
-		"@angular/core": "8.x - 13.x",
-		"@angular/forms": "8.x - 13.x",
+		"@angular/common": "8.x - 14.x",
+		"@angular/core": "8.x - 14.x",
```
Comment (rajao111) [inline projects/ngx-intl-tel-input/package.json:6]: @angular/forms

**Comment by kkachniarz220** [comment]
Accept this PR bro

**Comment by kevynsax** [comment]
Someone can please do a Fork. It is insane how much time this guy is taking to approve a simple PR

**Comment by guillermodelagala** [comment]
I'm using this fork for my purposes: [npm i ngx-intl-tel-input-gg](https://www.npmjs.com/package/ngx-intl-tel-input-gg)

**Comment by jorool** [comment]
The beauty of open-source projects....

**Comment by jorool** [comment]
@webcat12345 please merge

**Comment by mveelaid** [comment]
@webcat12345, It would be terrific if you could merge it anytime soon.

**Comment by itseramin** [comment]
@webcat12345 Please merge.

**Comment by dnapp** [comment]
@webcat12345 Could you please merge? Angular 15 was just released so it would be great if we could continue moving forward.

**Comment by dnlrbz** [comment]
@clempire1 @rajao111 Hi, are there any plans to support Angular 14? Please merge this PR, so that we could update our applications

**Comment by TofanIulian** [comment]
Hi, I noticed that there was another pull request that upgraded the repo to angular 14, [472](https://github.com/webcat12345/ngx-intl-tel-input/pull/472). Unfortunately, the upgrade was actually incomplete and the peerDependecies were overlooked which is why we still have incompatibility issues.

I've created a new PR [481](https://github.com/webcat12345/ngx-intl-tel-input/pull/481) to fix it.

@webcat12345, @nicolasleb, please take a look.

**Pull request closed without merging**